python/ray/tune/impl/tuner_internal.py

```diff
@@ -431,9 +446,11 @@ def _fit_resume(self, trainable) -> ExperimentAnalysis:
             **self._tuner_kwargs,
         }
         analysis = run(**args)
+        self.clear_remote_string_queue()
         return analysis
 
     def __getstate__(self):
+        self.clear_remote_string_queue()
```

Comment:
Hm, shouldn't we remove the string queue from the state dict instead? If we do this this means we clear the string queue anytime we pickle the state of the object. We only do this internally, but we should still try to avoid these footguns

Reply:
I.e.

```
state["_tuner_kwargs"].pop("_remote_string_queue", None)
```

Reply:
yeah! let me change it